pkgs/tools/security/argocd-vault-plugin/default.nix

```diff
@@ -13,9 +18,23 @@ buildGoModule rec {
 
   vendorHash = "sha256-0PrGrcS8Gx0cVImGrlmXlycFgWCTLjg2ISi0OhYoPpw=";
 
+  ldflags = [
+    "-X=github.com/argoproj-labs/argocd-vault-plugin/version.Version=v${version}"
+    "-X=github.com/argoproj-labs/argocd-vault-plugin/version.BuildDate=1970-01-01T00:00:00Z"
```

Comment:
why `1970-01-01T00:00:00Z` as BuildDate?

Reply:
would `builtins.currentTime` make sense here at all? Just curious.

Reply:
`1970-01-01T00:00:00Z` is the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time), it is the "zero" of Unix time basically.


`builtins.currentTime` is [impure](https://nixos.org/manual/nix/unstable/command-ref/conf-file#conf-pure-eval), using it would result in non-[reproducible builds](https://reproducible-builds.org/)

Reply:
blah, duh! Thx